### PR TITLE
Remove request params from cache key.

### PR DESCRIPTION
**Pull request by MasterSergius**
As we cache only HEAD and GET, we don't need request parameters anymore.

### CODE REVIEW

**Review by pika1989 — CHANGES_REQUESTED** [review]
Could you please add docstrings to the methods?

**Comment by MasterSergius** [comment]
> Could you please add docstrings to the methods?

Added, whenever possible, pls review.

**Review by pika1989 — APPROVED** [review]
LGTM

File: src/acachecontrol/cache.py [hunk 1]
```diff
@@ -31,37 +38,36 @@ def __init__(self, config={}):
             "cacheable_methods", ("HEAD", "GET")
```
Comment (andrii-ivaniuk) [inline src/acachecontrol/cache.py]: Let's define constant for cacheable methods to be consistent.

Comment (MasterSergius) [reply src/acachecontrol/cache.py]: Done

**Pull request merged**